gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri May 06 14:56:43 CEST 2016
+#Thu Jun 16 14:29:21 CEST 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
```

Comment:
Maybe Sha256 checksum [verification](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:verification) could be added?